reference/websockets/sails.io.js/socket.request.md

```diff
@@ -10,7 +10,30 @@ This function is very similar to `io.socket.get()`, `io.socket.post()`, etc. exc
 ### Usage
 
 ```js
-io.socket.request(options, function (data, jwr)){
+io.socket.request(url, reqData, function (resData, jwr)){
+  // ...
+  // jwr.headers
+  // jwr.statusCode
+  // jwr.body === data
+  // ...
+}, method);
+```
+
+
+### Example
+
+```javascript
+io.socket.request('/user/3/friends', {"friends":"4"}, function (resData, jwr)){
+  // ...
+  // jwr.headers
+  // jwr.statusCode
+  // jwr.body === data
+  // ...
+}, 'post');
+```
+
+```js
+io.socket._request(options, function (data, jwr)){
```

Comment:
`_request()` is deprecated, see: https://github.com/balderdashy/sails-generate-sails.io.js/blob/master/templates/sails.io.js#L924